Node Status Monitor/NodeJS/node_monitor.js

```diff
@@ -0,0 +1,197 @@
+/* imports*/
+import fetch from 'node-fetch';
+import fs from 'fs';
+import log from './utils/logger.js';
+import { get_token } from './api/auth.js';
+import config from './config/config.js';
+
+
+/* 
+variables:
+        Key                             Description
+    Only_changed_data :  if true get only changed node status, all node with status
+    
+functions:
+        Function Name                   Description
+    fetch_Data            :     function to fetch Node Details
+    process_node_status   :     function to process Node Details
+    write_data            :     function to write Node Details
+    read_old_data         :     function to read old Node Details
+    compare_data          :     function to compare  Node Details 
+    get_token             :     function to get Access token 
+    
+Files:
+        FileName                        Description
+    new_node_data.json    :      Json Format Result with Node's changed Status
+    old_node_data.json    :      Json Format Data Node's status Previous run
+    result.json         :      Json Format Data Node's status Current run  
+*/
+
+
+
+
+/* Gobal Variable */
+var only_changed_data = true; // if true get only changed node status, all node with status
+
+let nodes_detail_url = config.nodes_detail_url;
+let client_key = config.client_key;
+let client_secret = config.client_secret;
+// files
+let new_data_file = config.files.new_data_file;
+let old_data_file = config.files.old_data_file;
+let result_file = config.files.result_file;
+
+/* function call */
+get_node_details();
```

Comment:
No need to comment that this is a function call. Would also be more clear if placed after all method definitions.

Reply:
Changed!